src/bin/main.rs

```diff
@@ -1,12 +1,9 @@
 #[macro_use]
```

Comment:
I have a doubt here: doesn't this change require changes to the Cargo.toml file?
I have seen this in the Cargo.toml file for binaries. I guess that for the main binary it is not necessary, Am I right?
```toml
[[bin]]
name = "main"
test = false
doctest = true
bench = false
```

Reply:
The Cargo.toml configuration for binary crates is supported OOTB, when you use `main.rs` and `lib.rs`  files in your `src` directory. `cargo` knows how to execute this crate when you execute it.

Here is an I did when reading the book, building the `minigrep` solution:

https://www.github.com/EstebanBorai/the-rust-programming-language/tree/main/minigrep%2FCargo.toml

It's called "Separation of Concerns for Binary Projects".

When you want to have something else which doesn't follows this convention, then you can always overwrite this behavior using the configuration you provided above.